packages/api/src/router/review.ts

```diff
@@ -18,18 +20,30 @@ export const reviewRouter = {
           .optional(),
       }),
     )
-    .query(({ ctx, input }) => {
+    .query(async ({ ctx, input }) => {
       const { options } = input;
 
       const conditions = [
         options?.cycle && eq(Review.workTerm, options.cycle),
         options?.term && eq(Review.workEnvironment, options.term),
       ].filter(Boolean);
 
-      return ctx.db.query.Review.findMany({
+      const reviews = await ctx.db.query.Review.findMany({
         orderBy: desc(Review.id),
         where: conditions.length > 0 ? and(...conditions) : undefined,
       });
+
+      if (!input.search) {
+        return reviews;
+      }
+
+      const fuseOptions = {
```

Comment:
Can we maybe abstract the fuzzy searching out to a helper? I might implement a custom linting rule that makes sure that every function should be 4 or less lines of code 😡 /s

Reply:
will implement that on a separate branch